streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/JobWorker.java

```diff
@@ -1,160 +1,137 @@
 package org.ray.streaming.runtime.worker;
 
-import java.io.Serializable;
-import java.util.Map;
-
+import java.util.List;
 import org.ray.api.Ray;
+import org.ray.api.annotation.RayRemote;
 import org.ray.runtime.RayMultiWorkerNativeRuntime;
 import org.ray.runtime.functionmanager.JavaFunctionDescriptor;
-import org.ray.streaming.runtime.core.graph.ExecutionGraph;
-import org.ray.streaming.runtime.core.graph.ExecutionNode;
-import org.ray.streaming.runtime.core.graph.ExecutionNode.NodeType;
-import org.ray.streaming.runtime.core.graph.ExecutionTask;
+import org.ray.streaming.runtime.config.StreamingWorkerConfig;
+import org.ray.streaming.runtime.config.types.TransferChannelType;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionEdge;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionVertex;
 import org.ray.streaming.runtime.core.processor.OneInputProcessor;
 import org.ray.streaming.runtime.core.processor.ProcessBuilder;
 import org.ray.streaming.runtime.core.processor.SourceProcessor;
 import org.ray.streaming.runtime.core.processor.StreamProcessor;
+import org.ray.streaming.runtime.core.processor.TwoInputProcessor;
 import org.ray.streaming.runtime.transfer.TransferHandler;
 import org.ray.streaming.runtime.util.EnvUtil;
-import org.ray.streaming.runtime.worker.context.WorkerContext;
+import org.ray.streaming.runtime.worker.context.JobWorkerContext;
 import org.ray.streaming.runtime.worker.tasks.OneInputStreamTask;
 import org.ray.streaming.runtime.worker.tasks.SourceStreamTask;
 import org.ray.streaming.runtime.worker.tasks.StreamTask;
-import org.ray.streaming.util.Config;
+import org.ray.streaming.runtime.worker.tasks.TwoInputStreamTask;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * The stream job worker, it is a ray actor.
+ * The streaming worker implementation class, it is ray actor.
  */
-public class JobWorker implements Serializable {
-  private static final Logger LOGGER = LoggerFactory.getLogger(JobWorker.class);
+@RayRemote
```

Comment:
Remove this annotation. It's deprecated.

Reply:
Why remove? JobWorker is Actor.

Reply:
new ray api doesn't need  this. Please check ray java ut for example

Reply:
Thanks. It has been removed.